libs/@blockprotocol/core/src/types.ts

```diff
@@ -40,6 +40,8 @@ export type BlockMetadataRepository =
     }
   | string;
 
+type VersionedUrl = `${string}v/${number}`;
```

Comment:
`core` has very few dependencies and I would like to keep it that way (especially when we might have other services that don't need to bring in potential build complications from the type system package)